### PR TITLE
logging set up

### LINKED ISSUE #12: Longing
#### Description
Setting up logging in a Node.js project using a library like Morgan and  configuring logging levels and 
output destinations is a common task.

#### Tasks

- [ ] Set up a logging library 
- [ ] Configure logging levels and output destinations

#### Longing libaries

- morgan
- @types/morgan

#### Configure logging levels and output destinastions

- information or requirement 
- debug
- error

### PR DESCRIPTION
**Pull request by Dawaic6**
### What does this PR do?
This PR establishes set up logging library and configuration of logging level  and output them into the file called output.log 
### Description of Task to be completed?
 -Set up a logging library
- Configure logging levels and output destinations
### How should this be manually tested?
after  starting  server you can send request  using  postman , swagger ,.. after check in the output.log 
### Run Commands
- for start server : `npm run dev`
### related issue
#12

### CODE REVIEW

File: src/output.log [hunk 1]
```diff
@@ -0,0 +1,2 @@
+GET / 200 - - 3.080 ms Thu, 25 Apr 2024 17:13:52 GMT this morgan
+GET / 200 - - 2.840 ms Fri, 26 Apr 2024 10:50:00 GMT this morgan
```
Comment (Habinezajanvier) [inline src/output.log]: Can you put this file in your `gitignore`

**Review by Habinezajanvier — CHANGES_REQUESTED** [review]

File: src/index.ts [hunk 2]
```diff
@@ -6,12 +6,28 @@ import swaggerSpec from "./docs/swaggerconfig";
...
+let logStream = fs.createWriteStream(path.join(__dirname,'output.log'), {
+  flags: 'a'
+})
+
+morgan.token('type', function (req,res){
+return req.headers['content-type']
+})
+
+app.use(cors());
+app.use(morgan(":method :url :status :res[content-lenght] - :response-time ms :date[web] :type",{ stream: logStream}));
```
Comment (Habinezajanvier) [inline src/index.ts]: You can only do `combined` than adding `":method :url :status :res[content-lenght] - :response-time ms :date[web] :type"`

**Review by Habinezajanvier — CHANGES_REQUESTED** [review]

**Review by ambroisegithub — COMMENTED** [review]
That's  look Good!!!

File: src/index.ts (hunk 2 above)
Comment (Dawaic6) [reply src/index.ts]: its done

**Review by wayneleon1 — APPROVED** [review]

**Review by Habinezajanvier — APPROVED** [review]
👍

**Pull request merged**